packages/openapi3/package.json

```diff
@@ -16,6 +16,9 @@
   "keywords": [
     "typespec"
   ],
+  "bin": {
+    "openapi3-to-tsp": "cmd/openapi3-to-tsp.js"
```

Comment:
related to comment above but naming this `tsp-openapi3` with `convert` command might be better?

Reply:
Love this - I didn't know `tsp-client` existed and like following a similar naming scheme.